sysutils/pfSense-pkg-apcupsd/Makefile

```diff
@@ -35,6 +35,10 @@ do-install:
 		${STAGEDIR}${PREFIX}/pkg
 	${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/apcupsd_status.php \
 		${STAGEDIR}${PREFIX}/www
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/widgets/widgets/apcupsd.widget.php \
+		${STAGEDIR}${PREFIX}/www/widgets/windgets
+	${INSTALL_DATA} ${FILESDIR}${PREFIX}/www/widgets/include/apcupsd_inc.inc \
+		${STAGEDIR}${PREFIX}/www/widgets/include
```

Comment:
`${STAGEDIR}${PREFIX}/www/widgets/include` must be created earlier as well

Reply:
Changes submitted